### DISCUSSION TITLE
Daily 07/06

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#549] [Daily Sprint]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué haré hoy?
- Impedimentos o dificultades

**Comment by BraianTroncoso** [comment]
Hoy tuve clases de Python y Js, y tengo más tarde clases de Java. Tambien ya estoy trabajando en las issues que me asigné.

**Comment by NataliaAlvarezIspc** [comment]
Ayer empecé con la tarea que me había asignado de back, pude hacer la parte de angular.
Hoy estoy haciendo la parte de django. Por la noche tengo clase de Inglés, así que también tengo que dedicar un tiempo a hacer la tarea para la clase de hoy.

**Comment by rpgrca** [comment]
Hoy voy a estar en la clase de programación para ver si toman lista a los que no tuvimos ayer. Ayer también terminé la parte de selección, reporté un par de cosas que quedaron colgadas para no olvidarme después de arreglar eso. Hoy voy a intentar hacer lo de ventas, que se graben las ventas y que aparezcan listadas tanto en la parte de administración y la parte del cliente. Espero terminar con todo eso hoy y poder empezar a hacer otras vistas desde mañana.
